src/frontend/parser.mly

```diff
@@ -225,6 +245,11 @@ top_var_decl:
                        is_global= true};
        smeta= {loc=Location_span.of_positions_exn $startpos $endpos}}
     }
+  (*
+  | top_var_type OFFSET UNREACHABLE option(dims) SEMICOLON
+    { raise (Failure "This should be unreachable; the UNREACHABLE token should \
+                      never be produced") }
+   *)
```

Comment:
Do you need this commented out code?

Reply:
Whoops, fixed this with commit revise and push --force - turns out that doesnt work. fixed now